src/js/flows/submitSearch.js

```diff
@@ -51,3 +51,13 @@ export default function submitSearch(
     }
   }
 }
+
+function shouldClearTable(args, prev) {
+  let duration = ([from, to]) => to - from
+
+  return !(
+    args.tableProgram === prev.tableProgram &&
+    args.spaceId === prev.spaceId &&
+    duration(args.span) === duration(prev.span)
```

Comment:
Will you also add an `if this space is ingesting` check. I think if the space is not ingesting and I hit "enter", I'd want to see it clear and re-populate. What do you think?